src/app/rundown/components/segment/segment.component.html

```diff
@@ -1,14 +1,29 @@
-<div class="segment" [ngClass]="{'segment-on-air': segment.isOnAir}">
-  <span class="title">
-    {{segment.name}}
-  </span>
-  <span *ngIf="segment.isOnAir" class="on-air">
-    ON AIR
-  </span>
-  <span *ngIf="segment.isNext" class="next">
-    NEXT
-  </span>
-  <div class="parts">
-    <sofie-part *ngFor="let part of segment.parts" [part]="part" [isRundownActive]="isRundownActive" (partSelectedAsNextEvent)="emitSetNextEvent($event)"></sofie-part>
+<div class="c-segment" [class.c-segment--on-air]="segment.isOnAir" [class.c-segment--set-as-next]="segment.isNext">
+  <div class="c-segment__header">
+    <div class="c-segment__header-toolbar">
+      <span>123</span>
+      <span style="margin-left: 50px;">00:30</span>
+    </div>
+    <div class="c-segment__title">
```

Comment:
Any reason for this not being a span?

Reply:
Not really. We are just relying on the blocking nature of the div.
But the span can be made blocking as well. Do you want us to change that?